pkg/webhook/server.go

```diff
@@ -40,14 +43,16 @@ type PubSubClientConfig struct {
 // NewServer creates a new HTTP server implementation that will handle
 // receiving webhook payloads.
 func NewServer(ctx context.Context, cfg *Config, pubsubClientOpts ...option.ClientOption) (*Server, error) {
-	pubsub, err := NewPubSubMessenger(ctx, cfg.ProjectID, cfg.TopicID, pubsubClientOpts...)
+	pubsub, err := client.NewPubSubMessenger(ctx, cfg.ProjectID, cfg.TopicID, pubsubClientOpts...)
```

Comment:
Where is this coming from now?

Reply:
renamed the package to `clients`. It has BQ and Pubsub in there.